### DISCUSSION TITLE
Downloading Win11 Explorer

### DISCUSSION THREAD

**Discussion by Sadomina2** [zetaloop/ExplorerPatcher#40] [General]
Where or what name is used to get the latest, running copy of this file?
Whenever I click on it, I get a screen full of executables and build files.
I am not a coder or programer, just a user.
Thank you, in advance.
I am steve.domina@gmail.com located in Globe, Arizona, USA.

**Comment by zetaloop** [comment]
Sorry, but I think you are looking for the English version of ExplorerPatcher, right?  This is the Chinese community localization version, and the original version can be found at https://github.com/valinet/ExplorerPatcher.

By the way, you can download the executable file by going to the **Release** page and find the newest `ep_setup.exe`.

![Screenshot_20231010-082809.jpg](https://github.com/zetaloop/ExplorerPatcher/assets/36418285/52a777cf-b426-4fa0-b5a0-af84ea62fd20)

![Screenshot_20231010-082711.jpg](https://github.com/zetaloop/ExplorerPatcher/assets/36418285/8f7b0eef-a70f-4e77-9710-8d27e78c5da6)

**Comment by Sadomina2** [comment]
Thank you.. 

⁣Get TypeApp for Android ​
